src/components/patch/PatchContainer.tsx

```diff
@@ -55,9 +57,15 @@ interface State {
   selectedPatchTypes: string[];
 }
 
+interface paramsShape {
```

Comment:
interfaces are typically ThisCase. Is this interface describing the params passed from react router? a more descriptive variable name could help me as the reader determine what these params are for.

Reply:
thanks!